doc/tutorials/README.md

```diff
@@ -1,41 +1,41 @@
+* [Building mlpack from source](https://www.mlpack.org/doc/mlpack-git/doxygen/build.html)
+* [File Formats in mlpack](https://www.mlpack.org/doc/mlpack-git/doxygen/formatdoc.html)
+* [Matrices in mlpack](https://www.mlpack.org/doc/mlpack-git/doxygen/matrices.html)
+* [mlpack input and output](https://www.mlpack.org/doc/mlpack-git/doxygen/iodoc.html)
+* [mlpack timers](https://www.mlpack.org/doc/mlpack-git/doxygen/timer.html)
+* [Simple sample mlpack programs](https://www.mlpack.org/doc/mlpack-git/doxygen/sample.html)
 
 
 ### Method-specific tutorials
 
 These tutorials introduce the various methods mlpack offers, aimed at users who want to get started quickly. These tutorials start with simple examples and progress to complex, extensible uses.
 
-* [NeighborSearch tutorial (mlpack_knn / mlpack_kfn)](http://www.mlpack.org/docs/mlpack-git/doxygen/nstutorial.html)
-* [LinearRegression tutorial (mlpack_linear_regression)](http://www.mlpack.org/docs/mlpack-git/doxygen/lrtutorial.html)
-* [RangeSearch tutorial (mlpack_range_search)](http://www.mlpack.org/docs/mlpack-git/doxygen/rstutorial.html)
-* [Density Estimation Trees tutorial (mlpack_det)](http://www.mlpack.org/docs/mlpack-git/doxygen/dettutorial.html)
-* [K-Means tutorial (mlpack_kmeans)](http://www.mlpack.org/docs/mlpack-git/doxygen/kmtutorial.html)
-* [FastMKS tutorial (mlpack_fastmks)](http://www.mlpack.org/docs/mlpack-git/doxygen/fmkstutorial.html)
-* [Euclidean Minimum Spanning Trees tutorial (mlpack_emst)](http://www.mlpack.org/docs/mlpack-git/doxygen/emst_tutorial.html)
-* [Alternating Matrix Factorization Tutorial](http://www.mlpack.org/docs/mlpack-git/doxygen/amftutorial.html)
-* [Collaborative Filtering Tutorial](http://www.mlpack.org/docs/mlpack-git/doxygen/cftutorial.html)
-* [Conventional Neural Evolution Tutorial](http://www.mlpack.org/docs/mlpack-git/doxygen/cnetutorial.html)
+* [NeighborSearch tutorial (mlpack_knn / mlpack_kfn)](https://www.mlpack.org/doc/mlpack-git/doxygen/nstutorial.html)
+* [LinearRegression tutorial (mlpack_linear_regression)](https://www.mlpack.org/doc/mlpack-git/doxygen/lrtutorial.html)
+* [RangeSearch tutorial (mlpack_range_search)](https://www.mlpack.org/doc/mlpack-git/doxygen/rstutorial.html)
+* [Density Estimation Trees tutorial (mlpack_det)](https://www.mlpack.org/doc/mlpack-git/doxygen/dettutorial.html)
+* [K-Means tutorial (mlpack_kmeans)](https://www.mlpack.org/doc/mlpack-git/doxygen/kmtutorial.html)
+* [FastMKS tutorial (mlpack_fastmks)](https://www.mlpack.org/doc/mlpack-git/doxygen/fmkstutorial.html)
+* [Euclidean Minimum Spanning Trees tutorial (mlpack_emst)](https://www.mlpack.org/doc/mlpack-git/doxygen/emst_tutorial.html)
+* [Alternating Matrix Factorization Tutorial](https://www.mlpack.org/doc/mlpack-git/doxygen/amftutorial.html)
+* [Collaborative Filtering Tutorial](https://www.mlpack.org/doc/mlpack-git/doxygen/cftutorial.html)
+* [Conventional Neural Evolution Tutorial](https://www.mlpack.org/doc/mlpack-git/doxygen/cnetutorial.html)
```

Comment:
This link returns a 404.

Reply:
Yeah, that one needed to be removed since we removed the CNE tutorial.  Thanks!